app/controllers/registrations_controller.rb

```diff
@@ -3,7 +3,7 @@ class RegistrationsController < Devise::RegistrationsController
   private
 
   def sign_up_params
-    params.require(:user).permit(:name, :email, :password, :password_confirmation)
+    params.require(:user).permit(:name, :email, :password, :password_confirmation).merge(:locale => I18n.locale)
```

Comment:
> The point is to not let Arabic users change the global locale for all users.

What do you mean by 'all users'? how can a user change the 'global locale' for all the users? 

I think that understand how `around_action`works but I have difficulties to see its practical applicability in this (or other) example...